parodos-model-api/src/main/java/com/redhat/parodos/workflow/utils/WorkContextUtils.java

```diff
@@ -35,17 +35,33 @@ public class WorkContextUtils {
 	private WorkContextUtils() {
 	}
 
+	/**
+	 * method to get project id from workContext
+	 * @param workContext
+	 * @return project id
+	 */
 	public static String getProjectId(WorkContext workContext) {
```

Comment:
Can we move this method to WorkContext? Why do we need to have an external method?

Reply:
I will create a ticket in our jiraboard for this issue

Reply:
https://issues.redhat.com/browse/FLPATH-289